lib/src/rust/api.dart

```diff
@@ -53,9 +54,6 @@ abstract class AppSettings implements RustOpaqueInterface {}
 // Rust type: RustOpaqueMoi<flutter_rust_bridge::for_generated::RustAutoOpaqueInner<ThemeMode>>
 abstract class ThemeMode implements RustOpaqueInterface {}
 
-// Rust type: RustOpaqueMoi<flutter_rust_bridge::for_generated::RustAutoOpaqueInner<WhitenoiseError>>
-abstract class WhitenoiseError implements RustOpaqueInterface {}
```

Comment:
@erskingardner  I think it was more intuitive to have `WhitenoiseError` here... cause it is a class that is more general and used broadly, not just for groups. But this seems to be intentional, is there a reason to move it there?

Reply:
it was autogenerated.